pursuing-toxic-agenda.html

```diff
@@ -581,7 +581,7 @@ <h2 id="h.va0wuax2pbgz">Environmental Data Justice</h2>
             <p><a href="#ftnt_ref24" id="ftnt24">[24]</a> U.S. EPA, &ldquo;Environmental Justice Fact Sheet: National Environmental Justice Advisory Council,&rdquo; January, 2010, <a href="https://swap.stanford.edu/20131015222711/http://www.epa.gov/environmentaljustice/resources/publications/factsheets/fact-sheet-nejac-2009.pdf">https://swap.stanford.edu/20131015222711/http://www.epa.gov/environmentaljustice/resources/publications/factsheets/fact-sheet-nejac-2009.pdf</a>.</p>
         </div>
         <div>
-            <p><a href="#ftnt_ref25" id="ftnt25">[25]</a> Lindsey Dillon, <a href="#id.qskbsdioqlsb">interview with Mustafa </a><a href="#id.qskbsdioqlsb">Ali</a>, August 8, 2017. For example, NEJAC was instrumental in pushing the EPA brownfields program to incorporate economic equity considerations, such as the Brownfields Job Training Program. Many brownfields (or industrial properties requiring toxic cleanup) are located in poor communities that struggle with unemployment, and are at risk of being displaced by gentrification that often follows from environmental cleanup and greening projects. The Brownfield Job Training Program is an attempt to bring some of the benefits of environmental cleanup to the communities that have been most impacted by toxic waste. </p>
+            <p><a href="#ftnt_ref25" id="ftnt25">[25]</a> Lindsey Dillon, <a href="/mustafa-ali-interview/">interview with Mustafa </a><a href="#id.qskbsdioqlsb">Ali</a>, August 8, 2017. For example, NEJAC was instrumental in pushing the EPA brownfields program to incorporate economic equity considerations, such as the Brownfields Job Training Program. Many brownfields (or industrial properties requiring toxic cleanup) are located in poor communities that struggle with unemployment, and are at risk of being displaced by gentrification that often follows from environmental cleanup and greening projects. The Brownfield Job Training Program is an attempt to bring some of the benefits of environmental cleanup to the communities that have been most impacted by toxic waste. </p>
```

Comment:
Can we change this to:

`<a href="/mustafa-ali-interview/">interview with Mustafa Ali</a>`